packages/components/addon/components/hds/page-header/index.hbs

```diff
@@ -0,0 +1,30 @@
+{{!
+Copyright (c) HashiCorp, Inc.
+SPDX-License-Identifier: MPL-2.0
+}}
+<div class="hds-page-header" ...attributes>
+  {{yield (hash Breadcrumb=(component "hds/yield"))}}
+  <header class="hds-page-header__wrapper">
+    {{yield (hash IconTile=(component "hds/icon-tile" size="medium"))}}
+    <div class="hds-page-header__main">
+      <div class="hds-page-header__content">
+        <div class="hds-page-header__heading">
+          <h1 class="hds-page-header__heading-content hds-typography-display-500">
+            {{this.title}}
+          </h1>
+          {{yield (hash Badges=(component "hds/page-header/badges"))}}
+        </div>
+        <div class="hds-page-header__metadata">
+          {{yield
+            (hash
+              Subtitle=(component "hds/page-header/subtitle")
```

Comment:
I was considering that, and upon further testing it _seems_ like it would make sense to use the same strategy as the rest of the yielded content and not pass the title as an argument. This is also the method used in the `Modal` and `Flyout`, and will make adoption easier. I'll make some adjustments and update the hierarchy.